lessons/intro/cython/index.md

```diff
@@ -625,6 +625,11 @@ directions[maze >= 0] = b' '  # Python level, using b' '
 directions[1, 2] == ord('x')  # C level, using char
 ```
 
+> [note]
+> Použití `matrix[a, b]` je v Cythonu rychlejší než `matrix[a][b]`, protože se
+> uvnitř dějí jiné věci.
```

Comment:
Jaké jiné věci? Myslím že když už se to zmiňuje, tak by tu měl být buď aspoň nástin vysvětlení, nebo bychom měli říct kde ty rozdíly hledat (třeba `– viz anotace u vygenerovaného kódu`).

Reply:
Tam jsem to objevil. Jde o to, že u dvourozměrného ndarray je definovaný přístup přes `[a, b], ale pokud se použije nejprve `[a]`, převede se výsledek an pythonnní objekt a na něm se volá `[b]`. Klidně to později reformuluju.